src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0"?>
+<xsl:stylesheet
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  exclude-result-prefixes="xsl"
+  version="1.1">
+
+  <xsl:output method="xml" indent="yes" encoding="UTF-8" />
+
+  <xsl:template match="para">
+    <p>
+      <xsl:apply-templates />
+    </p>
+  </xsl:template>
+
+  <xsl:template match="b">
+    <strong>
+      <xsl:apply-templates />
+    </strong>
+  </xsl:template>
```

Comment:
differences between <strong> and <b> is that: <strong> tags tell crawlers to treat the inner text as **important**, while <b> is simply a style tag. [http://www.html-5-tutorial.com/strong-and-b-elements.htm](http://www.html-5-tutorial.com/strong-and-b-elements.htm)

---
In reply to: [58978110](https://github.com/dotnet/docfx/pull/249#discussion_r58978110) [](ancestors = 58978110)